presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -758,7 +758,7 @@ protected Object visitArithmeticBinary(ArithmeticBinaryExpression node, Object c
         @Override
         protected Object visitComparisonExpression(ComparisonExpression node, Object context)
         {
-            ComparisonExpressionType type = node.getType();
+            ComparisonExpressionType type = node.getOperator();
```

Comment:
can you please also change `ComparisonExpressionType` to `ComparisonExpressionOperator` or `ComparisonExpression.Operator`?